src/components/elements/CurateIndividual/ReciterTabs.tsx

```diff
@@ -95,26 +92,30 @@ const ReciterTabs = ({ reciterData, fullName, fetchOriginalData }: { reciterData
   const onUndo = () => { }
   return (
     <>
-      {/* <p className={styles.addPubmade}>Add New record: <Badge text="primary" bg="white" className="" onClick={(e) =>{ setPubKey(!pubKey); setKey("AddPub") }}>pubMade</Badge></p>  */}
       <Tabs
         id="controlled-tab-example"
         activeKey={key}
         onSelect={(k) => onTabChange (k)}
-        className={`my-3 ${styles.tabsContainer}`}
+        className={`${styles.tabsContainer}`}
       > {
           filteredData.map((tabData: any, index: number) => {
             return (
               <Tab
                 eventKey={tabData.value}
                 key={tabData.value}
+                className={tabData.value === "AddPub" ? `${styles.curateTabsMainPub}` : `${styles.curateTabsMain}`}
                 disabled={tabData.value === "addNewRecord" ? true : false}
                 tabClassName={tabData.value === "addNewRecord" ? `${styles.tabDisplayNone}` : tabData.value === "AddPub" ? `${styles.pubMadeTab}` : `${styles.tabsMain}`}
                 title={
                   <div className={key === tabData.value ? `${styles.activeTab} ${styles.tabTitle}` : styles.tabTitle}>{tabData.name} {tabData.value === "addNewRecord" || tabData.value === "AddPub" ? "" : <div className={key === tabData.value ? `${styles.active} ${styles.circle}` : styles.circle}><span className={key === tabData.value ? `${styles.activeCount} ${styles.count}` : styles.count}>{tabData.count}</span></div>}</div>
                 }>
                 {
                   key === "AddPub" ?
-                    <TabAddPublication onReject={() => addPub()} onUndo={() => onUndo()} reciterData={reciterData} />
+                    <TabAddPublication 
+                    onReject={() => addPub()}
```

Comment:
is there a reason we are passing callback function and not directly like onReject={addPub} ?

Reply:
Reverted my code here as it is no longer used. Otherwise, the rest of the code has been already there.

Reply:
TabAddPublication is a component added by you right? so please update the way functions are passed, as the current way will create a new functions and would be inefficient. You can refer to examples here or how it's done in the rest of the code: https://bobbyhadz.com/blog/react-pass-function-as-prop